djangoplicity/archives/contrib/satchmo/freeorder/models.py

```diff
@@ -23,6 +22,12 @@
 import string
 import random
 
+import django
+if django.VERSION >= (2, 2):
```

Comment:
Same comment as before, `django.core.urlresolvers` was removed in 2.0.

Reply:
@danialeresbar As Francisco pointed out, this still needs to be updated.